### PR TITLE
[bitnami/symfony] Add VIB tests

### PR DESCRIPTION
**Pull request by migruiz4**
### Description of the change

The main objective of this PR is to publish our Bitnami Symfony container using VMware Image Builder. In order to do that, several changes are included:

- Increasing the existing test coverage of the asset by adding Goss tests.
- Update verify and publish VIB pipeline's definitions.

### Benefits

- Ensuring higher quality of the container catalog.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could introduce additional flakiness to the CI/CD.

### Applicable issues

NA

### CODE REVIEW

**Review by FraPazGal — COMMENTED** [review]
Hi @migruiz4, I have added a couple of comments related to the PHP tests added here, please check them when you can.

File: .vib/symfony/goss/vars.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - mysql
+files:
+  - paths:
+      - /opt/bitnami/symfony/skeleton/bin/console
+      - /opt/bitnami/symfony/skeleton/bin/phpunit
+directories:
+  - mode: "0775"
+    paths:
+      - /app
+      - /opt/bitnami/symfony/skeleton
```
[the remark this answers is not in the document]
Comment (migruiz4) [reply .vib/symfony/goss/vars.yaml:15]: Done 👍

File: .vib/common/goss/templates/check-php-fpm.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - /^upload_tmp_dir = \/opt\/bitnami\/php\/tmp/
+      - /^session.save_path = \/opt\/bitnami\/php\/var\/run\/session/
+      - /^error_log = \/dev\/stderr/
+  /opt/bitnami/php/etc/php-fpm.d/www.conf:
+    exists: true
+    contains:
+      - /^listen = \/opt\/bitnami\/php\/var\/run\/www.sock/
+  /opt/bitnami/php/logs/php-fpm.log:
+    filetype: symlink
+    exists: true
```
Comment (FraPazGal) [inline .vib/common/goss/templates/check-php-fpm.yaml:17]: Nitpick: we normally add the `linked-to` attribute, though I don't have any strong thoughts about it.

Comment (migruiz4) [reply .vib/common/goss/templates/check-php-fpm.yaml:16]: Nice tip! I didn't know about that attribute.

Updated the test adding it.

File: .vib/symfony/goss/vars.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+binaries:
+  - php
+  - composer
+  - gosu
```
Comment (FraPazGal) [inline .vib/symfony/goss/vars.yaml:4]: One last comment, could you please remove `gosu` from this list. Yesterday we merged https://github.com/bitnami/containers/pull/30862 removing this check from the catalog after the tool was internally removed.

Besides that, I'll approve the PR once the VIB triggers are also removed.

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**